tests/rados/test_node_drain_customer_bug.py

```diff
@@ -0,0 +1,292 @@
+    out = rados_obj.run_ceph_command(cmd=cmd_host_ls)
+    log.debug(f"The hosts in the cluster before starting the test are - {out}")
+
+    mgr_host_object_list = []
+    for node in ceph_nodes:
+        if node.role == "mgr":
+            mgr_host_object_list.append(node)
+            log.debug(f"The mgr host node is{node.hostname}")
+
+    mgr_daemon_list = mgr_obj.get_mgr_daemon_list()
+    log.debug(f"The MGR daemons list are -{mgr_daemon_list}")
+
+    log_lines = [
+        "mgr load Traceback",
+        "TypeError: __init__() got an unexpected keyword argument 'original_weight'",
+    ]
+
+    ceph_version = rados_obj.run_ceph_command(cmd="ceph version")
+    log.info(f"Current version on the cluster : {ceph_version}")
+    match_str = re.match(
+        r"ceph version (\d+)\.(\d+)\.(\d+)-(\d+)", ceph_version["version"]
+    )
+    major, minor, patch, build = match_str.groups()
+    bug_exists = False
+    exceptional_flag = False
+    if int(major) < 18:
+        bug_exists = True
+    elif int(major) == 18 and int(minor) < 2:
+        bug_exists = True
+    elif int(major) == 18 and int(minor) == 2 and int(patch) < 1:
+        bug_exists = True
+    elif int(major) == 18 and int(minor) == 2 and int(patch) == 1 and int(build) < 234:
+        bug_exists = True
+
+    if bug_exists:
+        log.info(f"The bug exists and  the ceph version is - {ceph_version}")
+    else:
+        log.info(f"The bug not exists and the ceph version is - {ceph_version}")
+
+    # Check the labels
+    cmd_host_ls = "ceph orch host ls"
+    out = rados_obj.run_ceph_command(cmd=cmd_host_ls)
+    drain_host = None
+    for node in out:
+        if "_no_schedule" in node["labels"]:
+            drain_host = node["hostname"]
+            break
+    if drain_host is None:
+        osd_hosts = rados_obj.get_osd_hosts()
+        log.info(f"The osd nodes list are-{osd_hosts}")
+        drain_host = random.choice(osd_hosts)
+
+    init_time, _ = installer.exec_command(cmd="sudo date '+%Y-%m-%d %H:%M:%S'")
+    log.info(f"The test execution  started at - {init_time} ")
+    try:
+        osd_count_before_test = get_node_osd_list(rados_obj, ceph_nodes, drain_host)
+        log.info(
+            f"The OSDs in the drain node before starting the test - {osd_count_before_test} "
+        )
+        mgr_daemon.start()
+        service_obj.remove_custom_host(host_node_name=drain_host)
+        time.sleep(300)
+        end_time, _ = installer.exec_command(cmd="sudo date '+%Y-%m-%d %H:%M:%S'")
+        log.info(f"The test execution  ends at - {end_time}")
+        if not verify_mgr_traceback_log(
+            rados_obj=rados_obj,
+            start_time=init_time,
+            end_time=end_time,
+            mgr_daemon_list=mgr_daemon_list,
+            mgr_host_object_list=mgr_host_object_list,
+            lines=log_lines,
+        ):
+            log.error("Traceback messages are noticed in logs")
+            return 1
+        log.info(
```

Comment:
Add print statement saying that no error snippets were found in the mgr logs.

Reply:
I have included the message.